source/sass/buyers-guide/mixins/draft-product.scss

```diff
@@ -0,0 +1,16 @@
+// product draft rules
+@mixin draft-product {
```

Comment:
If there's only one version of `.draft`, for the SCSS part I think we can simply have rule set for `.draft` (without the mixin). 

```scss
.draft::before {
    content: 'This is a draft.';
    color: #ffa7a7;
    font-weight: bold;
    font-size: 120%;
    font-family: 'Zilla Slab';
    z-index: 1;
    position: absolute;
    top: 0;
    left: 0;
    right: 0;
    text-align: center;
}
```
Then we can simply apply this classname `.draft` to wrapper element of our choice (e.g., `<div class="product-header draft" ...>`). This `<div>` will auto pick up the rules above as it has `draft` as its class (we don't need to  do `@include` or `@import` multiple times)